pose-detection/src/calculators/keypoints_one_euro_filter.ts

```diff
@@ -47,16 +47,28 @@ export class KeypointsOneEuroFilter implements KeypointsFilter {
     // Initialize filters once.
     this.initializeFiltersIfEmpty(keypoints);
 
+    // Get value scale as inverse value of the object scale.
+    // If value is too small smoothing will be disabled and keypoints will be
+    // returned as is.
+    let valueScale = 1;
+    if (this.config.minAllowedObjectScale != null) {
```

Comment:
disableValueScaling should be used here

Reply:
This change was temporary and is now added and used in Add 3D keypoints output to TFJS blazepose commit